runtime/v2/shim/util_windows.go

```diff
@@ -28,3 +36,26 @@ func getSysProcAttr() *syscall.SysProcAttr {
 func SetScore(pid int) error {
 	return nil
 }
+
+// AbstractAddress returns an abstract npipe address
+func AbstractAddress(ctx context.Context, id string) (string, error) {
```

Comment:
that name doesn't make sense on windows.

Could we rename to just `SocketAddress` or `AddressPath` ? Or anything without `Abstract` in it  :)